shortcodes/ucf-pegasus-list-shortcode.php

```diff
@@ -25,11 +25,12 @@ public static function callback( $atts ) {
 
 			$layout = $atts['layout'];
 			$feed_url = $atts['feed_url'];
-			$limit = $atts['limit'];
+			$limit = isset( $atts['limit'] ) ? $atts['limit'] : 5;
```

Comment:
You shouldn't need to set a fallback `$limit` value of 5 here, since you have a default value set in `UCF_Pegasus_List_Config::$option_defaults`.  Same with `$offset`, if you add a default to `UCF_Pegasus_List_Config::$option_defaults`.